### PR TITLE
Deployed Documentation for the Guides

### PR DESCRIPTION
**Pull request by sonny-mo**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

Hi there, ✨

There was some issue with the documentation being hard to follow referring to #180. I thought a temporary fix might help for the time being. 

Here's the [deployed documentation](https://docs.contour.so/amzn/selling-partner-api-docs)

@ganeshkrishnan1 Maybe you might find this somewhat useful? I haven't been able to update all the links but you should be able to edit and reorder things around.

If there's more enthusiasm around this, I (or someone else) could migrate the references documentation in the repository as well.

The docs are deployed on a web app I made while on break from school; it has many bugs for the time being, but if you request changes, I'll make them ASAP (at latest day of) 😄.

### CODE REVIEW

**Comment by sonny-mo** [comment]
I know this doesn't fix all the issues with the broken links and stuff, but at least you can now edit them/mark them so that you don't stumble upon them everytime.

**Comment by seanevan** [comment]
Hi @sonny-mo,

We certainly appreciate the effort, and your deployed documentation is super clean to read!

Unfortunately we can't accept a PR that directs developers to a site that is not maintained by one of our contributers. We are working to deploy a similar experience, so please stay tuned.

Thanks,
Sean Evans
Manager, Support Engineering
